noir-projects/aztec-nr/aztec/src/utils/bytes.nr

```diff
@@ -78,15 +76,15 @@ mod test {
             1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24,
             25, 26, 27, 28, 29, 30, 31,
         ];
```

Comment:
I removed turbofish syntax in all the tests below as it makes it more explicit what info a dev needs to provide manually and what the compiler is able to derive on its own.

(A dev needs to provide only the number of bytes when converting from fields to bytes as the fields might contain less than `num_fields*31` bytes of info).